### ISSUE TITLE
azurePipeline - integration with hPublishTestResults@2  : Error : '<', hexadecimal value 0x3C, is an invalid attribute character.

### ISSUE THREAD

**Issue by rayatus** [abaplint/abaplint#2833]
I'm trying to integrate abaplint via Azure Build Pipelines but when analyzing results via [PublishTestResults@2](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/publish-test-results-v2?view=azure-pipelines&tabs=trx%2Ctrxattachments%2Cyaml) i'm having following "Warning" causing PublishTestResults@2 not being able to "read" them.

`##[warning]Failed to read /home/vsts/work/1/s/report.xml. Error : '<', hexadecimal value 0x3C, is an invalid attribute character. Line 130, position 77..` 
![image](https://user-images.githubusercontent.com/12736990/210869849-71849c43-cf83-4ec5-84cf-54c44d53de5f.png)

having a look at that result file (line 130, pos 77)

![image](https://user-images.githubusercontent.com/12736990/210869608-f7463173-fd18-4cc3-9af3-84dfd9e8745f.png)

abaplint is executed as follows

```bash 
abaplint -f standard --outformat junit --outfile report.xml || true
```
as code is propietary, I've prepared a similar example so it could be tested if necessary.

![image](https://user-images.githubusercontent.com/12736990/210871453-76c68bc1-1d35-4d6d-93f4-9b905c2fbbb5.png)

```abap
REPORT zfoobar.

CLASS lcl_foo DEFINITION FINAL.

  PUBLIC SECTION.
    METHODS error_method.
ENDCLASS.

CLASS lcl_foo IMPLEMENTATION.
  METHOD error_method.
    FIELD-SYMBOLS <params> TYPE data.
  ENDMETHOD.
ENDCLASS.
```

**Comment by rayatus** [comment]
I'd say that '<' is not accepted in FAILURE-MESSAGE property, so **maybe**:

- escaping var names in that section?
- if not... maybe avoid showing them in case they contains characters that would need to be escaped? 
- others?

**Comment by rayatus** [comment]
it turns out that it was already reported in #2582
